x/msgfees/client/cli/cli_test.go

```diff
@@ -59,6 +70,31 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.cfg = testutil.DefaultTestNetworkConfig()
 	s.cfg.TimeoutCommit = 500 * time.Millisecond
 	s.cfg.NumValidators = 1
+	s.GenerateAccountsWithKeyrings(1)
+
+	var genBalances []banktypes.Balance
+	for i := range s.accountAddresses {
+		genBalances = append(genBalances, banktypes.Balance{Address: s.accountAddresses[i].String(), Coins: sdk.NewCoins(
+			sdk.NewInt64Coin("nhash", 100_000_000), sdk.NewInt64Coin(s.cfg.BondDenom, 100_000_000),
+		).Sort()})
+	}
+	var bankGenState banktypes.GenesisState
+	bankGenState.Params = banktypes.DefaultParams()
+	bankGenState.Balances = genBalances
+	bankDataBz, err := s.cfg.Codec.MarshalJSON(&bankGenState)
+	s.Require().NoError(err, "should be able to marshal bank genesis state when setting up suite")
+	s.cfg.GenesisState[banktypes.ModuleName] = bankDataBz
+
+	var authData authtypes.GenesisState
+	var genAccounts []authtypes.GenesisAccount
+	authData.Params = authtypes.DefaultParams()
+	genAccounts = append(genAccounts, authtypes.NewBaseAccount(s.accountAddresses[0], nil, 3, 0))
+	accounts, err := authtypes.PackAccounts(genAccounts)
+	s.Require().NoError(err, "should be able to pack accounts for genesis state when setting up suite")
+	authData.Accounts = accounts
+	authDataBz, err := s.cfg.Codec.MarshalJSON(&authData)
+	s.Require().NoError(err, "should be able to marshal auth genesis state when setting up suite")
+	s.cfg.GenesisState[authtypes.ModuleName] = authDataBz
```

Comment:
Please use `testutil.MutateGenesisState` for this stuff. That way you're not stomping on entries that are already in there. See `x/marker/client/cli/cli_test.go` for an example.

Reply:
Ok, thanks for pointing this out.